Assets/Scripts/Managers/PlayerManager.cs

```diff
@@ -1,21 +1,16 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using Managers;
-using Player;
-using UnityEngine;
-using UnityEngine.Networking.PlayerConnection;
+﻿using UnityEngine;
 
 namespace Managers
 {
     public class PlayerManager : Singleton<PlayerManager>
     {
-        [SerializeField] private GameObject player;
+        [SerializeField] private GameObject _player;
```

Comment:
SerializedFields are normaly without underscore. Was already correct:)

Reply:
```suggestion
        [SerializeField] private GameObject player;
```

Reply:
oh ye sure :joy: my bad

Reply:
for a singleton I don't expect a serialize